src/Common/src/Interop/Linux/Interop.Libraries.cs

```diff
@@ -11,5 +11,6 @@ private static partial class Libraries
         internal const string INODE64SUFFIX = "";
 
         internal const string LibRt = "librt"; // POSIX Realtime Extensions library
+	internal const string LibCurl = "libcurl.so.4";             // Curl HTTP client library
```

Comment:
Why not libcurl.so.3?  What non-dev package do we install for libcurl4 on Ubuntu?